app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -11,38 +11,31 @@ def index
   end
 
   def create
-    @planning_application = PlanningApplication.create(full_planning_application_params)
-    if @planning_application.save!
-      @planning_application.create_policy_evaluation
-      render json: {"id": "#{@planning_application.reference}",
-                    "message": "Application created: id was #{@planning_application.reference}"}, status: 200
+    @plan_app = PlanningApplication.create(full_planning_application_params)
```

Comment:
Is there a reason you've abbreviated this variable name?

Reply:
Not a particularly logical reason... I will go back to it and reinstate the full name